### PR TITLE
Preparing run2 legacy v3

### PR DESCRIPTION
**Pull request by jedori0228**
Updating scripts and README for the next production

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,23 +63,20 @@ scram b -j 8
...
 export SCRAM_ARCH=slc6_amd64_gcc630
-scram p -n Run2Legacy_v2__CMSSW_9_4_10 CMSSW CMSSW_9_4_10
-cd Run2Legacy_v2__CMSSW_9_4_10/src
+scram p -n Run2Legacy_v3__CMSSW_9_4_13 CMSSW CMSSW_9_4_13
+cd Run2Legacy_v3__CMSSW_9_4_13/src
 cmsenv
 # crab setup
 source /cvmfs/cms.cern.ch/crab3/crab.sh
-# proxy
-voms-proxy-init --voms cms -valid 9999:00
```
Comment (jalmond) [inline README.md]: why remove this?

Comment (jedori0228) [reply README.md]: ah I thought proxy will be asked when submitting crab anyway, so not needed here

Comment (jalmond) [reply README.md]: Ok, i agree, but it wont be -valid 9999:00 if that ok with you

Comment (jedori0228) [reply README.md]: yep it needs to be only valid during the crab submission so I think it's okay (9999:00 is copied from my .bach_profile lol)

**Pull request merged**